lib/community/widgets/community_sidebar.dart

```diff
@@ -231,13 +231,16 @@ class CommunityModeratorList extends StatelessWidget {
                       mainAxisAlignment: MainAxisAlignment.start,
                       crossAxisAlignment: CrossAxisAlignment.start,
                       children: [
-                        Text(
-                          mods.moderator.displayName ?? mods.moderator.name,
-                          overflow: TextOverflow.ellipsis,
-                          maxLines: 1,
-                          style: const TextStyle(
-                            fontWeight: FontWeight.bold,
-                            fontSize: 16,
+                        SizedBox(
+                          width: MediaQuery.of(context).size.width * 0.55,
```

Comment:
Just curious, but where does the 0.55 come from?

Reply:
I just tweaked the percentage until it looked good. I figured since it was a percentage rather than a fixed value it should scale to other devices. What do you think?

I tried `FractionallySizedBox` first, since that's what the whole sidebar uses, but it didn't work.